src/Annotations/Serializer/HypothesisClientAnnotationNormalizer.php

```diff
@@ -169,4 +171,9 @@ public function supportsNormalization($data, $format = null) : bool
     {
         return $data instanceof Annotation;
     }
+
+    private function purify(string $text) : string
+    {
+        return $this->htmlPurifier->purify($text);
```

Comment:
I suspect we'll want to cache the results of these (IIRC the library doesn't do this).

This class is feeling a bit bloated now, knowing how to convert all the bits and also purify them. Can purification happen before/after?

Reply:
Purification happening after is a possibility. I implemented a solution where it was happening before but it wasn't optimal.

In order for it to happen after the `HypothesisClientAnnotationNormalizer` would return an unsanitized response and then we would need a class so step through each item in the `content` array. So, I would need 2 places where the structure of the response was known. Perhaps we can have a chat about this tomorrow.

Reply:
I've done a refactor here which means I can remove purify from this class. See what you think.